bin/sdl.cc

```diff
@@ -49,7 +49,7 @@ int main(int argc, char** argv)
         ("x,event_index"     , "specific event index to process", cxxopts::value<int>()->default_value("-1"))
         ("g,pdg_id"          , "The simhit pdgId match option (default = 0)", cxxopts::value<int>()->default_value("0"))
         ("v,verbose"         , "Verbose mode (0: no print, 1: only final timing, 2: object multiplitcity", cxxopts::value<int>()->default_value("0"))
-        ("w,write_ntuple"    , "Write Ntuple", cxxopts::value<int>()->default_value("1"))
```

Comment:
why do we don't want to write an ntuple in default?

Reply:
I rolled back to the previous default behavior. i.e. writing out the NTuple by default.

Reply:
Thanks!